pkg/agent/utils.go

```diff
@@ -250,17 +201,6 @@ func getLBRule(name string, port int) string {
           }`, port, name, name, port, name, port, name, port)
 }
 
-func getLBRules(name string, ports []int) string {
-	var buf bytes.Buffer
-	for index, port := range ports {
-		if index > 0 {
-			buf.WriteString(",\n")
-		}
-		buf.WriteString(getLBRule(name, port))
-	}
-	return buf.String()
-}
-
 func getProbe(port int) string {
```

Comment:
this

Reply:
getProbe function is.being removed in this new PR: https://github.com/Azure/AgentBaker/pull/308